src/index.ts

```diff
@@ -122,10 +127,22 @@ export function kebabCase<
         .join(joiner ?? "-") as JoinByCase<T, Joiner>);
 }
 
+export type SnakeCase<T extends string | readonly string[]> = JoinByCase<
```

Comment:
Maybe move all type utils to same file? Also could make exports more readable

Reply:
good idea 👌